interface/frontend/src/components/open-image/OpenDICOM.vue

```diff
@@ -124,6 +127,9 @@
       }
     },
     methods: {
+      areaSelectChange (newCoords) {
+        console.log('areaSelectChanged', JSON.stringify(newCoords))
```

Comment:
Out of interest, do we want to be logging directly to `console.log` here? :)

Reply:
The component I have implemented will be used on Segmentation page, but currently it [is disabled here](https://github.com/concept-to-clinic/concept-to-clinic/pull/249/files#diff-70d179422138a3042c2135a282065626R64), so there will be no logs. And once someone will start working on that page, he will enable the component at that line and handle user input at this line.